### ISSUE TITLE
Old DB data seems not properly updated by newer DB

### ISSUE THREAD

**Issue by yukimizutani** [orientechnologies/orientdb#8004]
## Steps to reproduce  
1. Create 2.2.29 DB
2. Update to 2.2.31
3. Try to stop DB

## Expected behavior  
Cleanly stops

## Actual behavior  
Stops with error saying it could not update a wal..
It does not seem to reproduce with a freshly created DB though.

```
com.orientechnologies.common.io.OIOException: Error during appending of new segment to the WAL
    at com.orientechnologies.orient.core.storage.impl.local.paginated.wal.ODiskWriteAheadLog.appendNewSegment(ODiskWriteAheadLog.java:1156)
    at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.makeFullCheckpoint(OAbstractPaginatedStorage.java:3707)
    at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.doClose(OAbstractPaginatedStorage.java:4397)
    at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.close(OAbstractPaginatedStorage.java:580)
...
Caused by: java.io.FileNotFoundException: ~/managementdb/hedgedb/hedgedb.2.wal (No such file or directory)
    at java.io.RandomAccessFile.open0(Native Method)
    at java.io.RandomAccessFile.open(RandomAccessFile.java:316)
    at java.io.RandomAccessFile.<init>(RandomAccessFile.java:243)
    at com.orientechnologies.orient.core.storage.impl.local.paginated.wal.OLogSegmentV2.getRndFile(OLogSegmentV2.java:397)
    at com.orientechnologies.orient.core.storage.impl.local.paginated.wal.OLogSegmentV2.init(OLogSegmentV2.java:441)
    at com.orientechnologies.orient.core.storage.impl.local.paginated.wal.ODiskWriteAheadLog.appendNewSegment(ODiskWriteAheadLog.java:627)
    at com.orientechnologies.orient.core.storage.impl.local.paginated.wal.ODiskWriteAheadLog.appendNewSegment(ODiskWriteAheadLog.java:1153)
    ... 20 common frames omitted
```

**Comment by laa** [comment]
@yukimizutani could you consistently reproduce this issue because I can not reproduce it.

**Comment by laa** [comment]
Hi @yukimizutani could you consistently reproduce this issue? Because I can not reproduce it.

**Comment by yukimizutani** [comment]
Hi @laa I do not reproduce this issue always. I get this issue time to time, and not sure why.

**Comment by laa** [comment]
Should be fixed in 3.0.x version.
